### PR TITLE
Adding global 'Run audit' action button to command action page

### PR DESCRIPTION
**Pull request by erica-banda-03**
Issue/change: https://github.com/kappnav/issues/issues/147

Changes: 
- New `Run Audit` button on command action page
- Support application name in notifications
- New method to open action modal 
- New message for inventory command action button

Kin added detailed commit messages about the changes he made.

### CODE REVIEW

File: src-web/actions/common.js [hunk 1]
```diff
@@ -188,6 +188,17 @@ export const openModal = (...args) => {
   )
 }
 
+export const openActionModal = (namespace, resourceName, actionName, actionDescription) => {
+  // var restApi = '/kappnav/resource/kappnav/execute/command/'+actionName+'?namespace=kappnav'
```
Comment (k8vance88) [inline src-web/actions/common.js]: Remove commented out code

File: src-web/components/kappnav/JobView.jsx [hunk 2]
```diff
@@ -75,27 +75,35 @@ class JobView extends Component {
...
+              sortColumn={this.state.sortColumn}
+              sortDirection={this.state.sortDirection}
+              handleSort={(e) => {
+                this.handleSort(e)
+              }}
+              pageNumber={this.state.pageNumber}
+              createNewModal={() => {
+                return (
+                  <div>
+                    <Button small iconDescription={msgs.get('run.audit')} onClick={() => openActionModal(document.documentElement.getAttribute('appnavConfigmapNamespace'), 'kappnav', 'app-nav-inventory', "Run Inventory Command Action")}>
```
Comment (k8vance88) [inline src-web/components/kappnav/JobView.jsx]: I think you need to translate "Run Inventory Command Action'

File: src-web/actions/common.js (hunk 1 above)
Comment (erica-banda-03) [reply src-web/actions/common.js]: I think the comment is there to show what the url would look like.

File: src-web/components/kappnav/JobView.jsx (hunk 2 above)
Comment (erica-banda-03) [reply src-web/components/kappnav/JobView.jsx]: I'll add the message so it can be translated.

File: src-web/actions/common.js (hunk 1 above)
Comment (k8vance88) [reply src-web/actions/common.js]: If that's the case can we remove all of the variables?

Comment (erica-banda-03) [reply src-web/actions/common.js]: Well, I may have been wrong. The namespace definitely not be the same all the time.  I removed the comment.

**Pull request merged**